ui/lib/ve-token.ts

```diff
@@ -4,45 +4,46 @@ import { veNationToken } from '../lib/config'
 import { useBalance, useContractRead, useContractWrite } from './use-wagmi'
 
 const contractParams = {
-  addressOrName: veNationToken,
-  contractInterface: VotingEscrow.abi,
+  address: veNationToken,
+  abi: VotingEscrow.abi,
 }
 
 export function useVeNationBalance(address: any) {
   return useBalance({
-    addressOrName: address,
+    address,
     token: veNationToken,
     watch: true,
-    enabled: address,
+    enabled: !!address,
   })
 }
 
 let gasLimits = {
   locked: 330000,
-  create_lock: 600000,
+  create_lock: 5000000,
```

Comment:
Let's keep the `create_lock` gas limit at 600,000 since that's the value needed for Mainnet - https://etherscan.io/tx/0x349ac19ff3dc79b964c7a6bbfc0e37a7f6aeb83eb6055604e221570f15c1f20a
```suggestion
  create_lock: 600000,
```